drv/hisi_sec.c

```diff
@@ -531,7 +531,6 @@ static void update_iv(struct wd_cipher_msg *msg)
 				msg->iv_bytes, msg->iv_bytes);
 		break;
 	case WD_CIPHER_CTR:
-		if (msg->iv_bytes >= AES_BLOCK_SIZE)
```

Comment:
You didn't explain why the conditional code should be removed.

Reply:
Due to has added the iv length checking in wd_cipher. so the ctr mode iv length checking is redundant.